packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2557,20 +2557,62 @@ describe('plugin-meetings', () => {
         it('should have #requestScreenShareFloor', () => {
           assert.exists(meeting.requestScreenShareFloor);
         });
+        let sandbox;
+
         beforeEach(() => {
           meeting.locusInfo.mediaShares = [{name: 'content', url: url1}];
           meeting.locusInfo.self = {url: url1};
           meeting.meetingRequest.changeMeetingFloor = sinon.stub().returns(Promise.resolve());
           meeting.mediaProperties.shareVideoTrack = {};
           meeting.mediaProperties.mediaDirection.sendShare = true;
           meeting.state = 'JOINED';
+          sandbox = sinon.createSandbox();
```

Comment:
why do we need a new sandbox? sinon has a default sandbox, you can call `sinon.restore()` on it in afterEach(), then you don't need a separate sandbox

Reply:
I was just following other examples. Will try without.

Reply:
I think those examples might be old code from the times when sinon did require explicitly creating a sandbox